lib/metadataTypes/definitions/Role.definition.js

```diff
@@ -76,7 +76,7 @@ module.exports = {
             template: false,
         },
         PermissionSets: {
-            retrieving: true,
+            retrieving: false, // added in retrieve method if not run for caching
```

Comment:
so reducing scope where not needed basically?

Reply:
No, it was indeed. we have a central helper method in our generic soap getter which retrieves the list of fields that are relevant for the retrieve request.
We COULD upgrade all definition files with a new attribute "caching" on each field or we use the workaround that im using here